1-3.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "[DRAFT] Unit 3: Business Case and Benefits"
+permalink: /curricula/1-3/
+ref: /curricula/1-3/
+lang: en
+github:
+  repository: w3c/wai-curricula
+  path: 1-3.md
+footer: >
+  <p>Work in Progress.</p>
+  <p><strong>Date:</strong> $date </p>
+  <p><strong>Editors:</strong>Daniel Montalvo and <a href="http://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Contributors: <a href="https://www.w3.org/WAI/EO/EOWG-members">EOWG Participants</a>. </p>
+  <p>Developed by the Education and Outreach Working Group (<a href="http://www.w3.org/WAI/EO/">EOWG</a>). Developed as part of the <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide Project</a> funded by the European Commission (EC) under the Horizon 2020 program (Grant Agreement 822245).</p>
+navigation:
+  previous: /curricula/1-2/
+  next: /curricula/1-4/
+---
+
+## Introduction
+{:.no-display}
+ 
+Courses based on this unit are expected to:
+* examine the rationale for organizations to address accessibility
+* present different testimonials from organizations that have addressed accessibility
+
+Note: Introduce different case studies for accessibility in organizations. Select them as a starting point for students to build their own business case arguments. Use the resource [The Business Case for Digital Accessibility](https://www.w3.org/wai/business-case/).
+
+## Learning Outcomes
+
+Students should be able to:
+* Identify accessibility as a driving force for innovation in products
+* Recognize accessibility as a discipline that promotes  inclusion
+* Explain how accessibility can extend market reach for organizations
+* Identify how to minimize legal risk for some organizations
+
+## Competencies
+
+The following is a list of skills required for this unit.
+
+Studentss
+: Basic &mdash; Basic understanding of computers and the Web.
+
+Instructors
+: Medium &mdash; Good understanding of accessibility and inclusion. Instructors should have experience in the types of business benefits resulting from accessibility.
+
+{% comment %}
+
+## Duration
+
+Approximately 1-2 hours depending on demonstrated examples.
+
+{% endcomment %}
+
+## Topics to Teach
+
+The following topics help achieve the learning outcomes. Adapt the sequence of topics, the activities you do, and the homework you assign for your particular students and context.
+
+{% include excol.html type="all" %}
+
+{% include excol.html type="start" %}
+
+### Topic: Accessibility as a Drive for Innovation
+
+{% include excol.html type="middle" %}
+
+Explain how accessibility helps organizations remove architectural, digital, and social barriers. Use the resource [Business Case for Digital Accessibility (Drive Innovation)](https://www.w3.org/WAI/business-case/#drive-innovation).
```

Comment:
Slight discomfort with "architectural" -- seems like scope creep and we should focus on digital accessibility here (comparisons and analogies are fine but here you as instructors to *explain* these aspects)